.github/workflows/deploy_production.yaml

```diff
@@ -0,0 +1,128 @@
+name: Build and Deploy Production
+run-name: Deploying to production
+on:
+  push:
+    branches:
+      - main
```

Comment:
We should probably track the commit in version control for each primary environment